test/freenet/support/MultiValueTableTest.java

```diff
@@ -319,13 +315,13 @@ public void testRemoveElement() {
 		MultiValueTable<Object, Object> methodMVTable = fillMultiValueTable(sampleObjects);
 		Object methodValue;
 		Iterator<Object> iter;
-		for(int i=0;i<sampleObjects.length;i++) {
-			iter = ((List<Object>)(sampleObjects[i][1])).iterator();
-			assertFalse(methodMVTable.removeElement(sampleObjects[i][0],new Object()));
-			while(iter.hasNext()) {
+		for (Object[] sampleObject : sampleObjects) {
```

Comment:
Cast `sampleObjects`, change the type of `sampleObject`, remove cast in line below.